.github/workflows/codegen_ci.yml

```diff
@@ -117,3 +117,25 @@ jobs:
       - name: Integration specs
         run: rspec
         working-directory: ./white_label
+
+  codegen-rubocop:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.2
+
+      - uses: actions/checkout@v2
+
+      - name: Copy Gemfile and dependencies
+        run: cp hearth/Gemfile hearth/hearth.gemspec hearth/VERSION codegen/
```

Comment:
Why do we need gemspec and version?

Reply:
Gemfile needs both to run the bundle install per our convo